src/modelmanager.cpp

```diff
@@ -470,6 +470,23 @@ StatusCode downloadModels(std::shared_ptr<FileSystem>& fs, ModelConfig& config,
     return StatusCode::OK;
 }
 
+Status ModelManager::cleanupModelTmpFiles(ModelConfig& config, std::shared_ptr<model_versions_t>& versionsToCleanup) {
+    auto lfstatus = StatusCode::OK;
+    if (versionsToCleanup->size() > 0) {
+        if (config.getLocalPath().compare(config.getBasePath())) {
+            LocalFileSystem lfs;
+            lfstatus = lfs.deleteFileFolder(config.getLocalPath());
+            if (lfstatus != StatusCode::OK) {
+                spdlog::error("Error occurred while deleting local copy of cloud model");
```

Comment:
Why you changed original error message here? It does not tell now where trash files are nor from which model they are.